common/network-common/src/main/java/org/apache/spark/network/sasl/SparkSaslServer.java

```diff
@@ -74,6 +76,9 @@
   private final SecretKeyHolder secretKeyHolder;
   private SaslServer saslServer;
 
+  private boolean isAesEnabled;
```

Comment:
Is it possible to remove isAesEnabled and use sparkSaslAES to check?

if(sparkSaslAES != null) {
}